shakedown/dcos/package.py

```diff
@@ -29,10 +28,10 @@ def _get_cosmos():
     """ Get an instance of Cosmos with the correct URL.
 
         :return: Cosmos instance
-        :rtype: cosmospackage.Cosmos
+        :rtype: packagemanager.Cosmos
```

Comment:
I'm not sure if this rtype is correct either... it appears to be `packagemanager.PackageManager`